backend/app/capabilities/embedding/pipeline.py

```
@@ -45,6 +46,11 @@
 MARKDOWN_HEADING_PATTERN = re.compile(
     r"^\s{0,3}(#{1,6})[ \t]+(.+?)(?:[ \t]+#+)?[ \t]*$"
 )
+PLAIN_SECTION_HEADING_PATTERN = re.compile(
+    r"^\s*(?P<title>第[〇零一二三四五六七八九十百千万两\d]+(?:编|部|篇|章|节)(?:[ \t]+.*)?|"
+    r"[一二三四五六七八九十百千万两\d]+[、.．][ \t]*\S.*|"
```

Ruff (0.16.1)

[warning] String contains ambiguous `．` (FULLWIDTH FULL STOP). Did you mean `.` (FULL STOP)?

(RUF001)